packages/grafana-test-utils/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "author": "Grafana Labs",
+  "license": "Apache-2.0",
+  "name": "@grafana/test-utils",
+  "version": "12.0.0-pre",
+  "private": true,
+  "description": "Grafana test utils & Mock API",
+  "keywords": [
+    "grafana",
+    "typescript",
+    "msw",
+    "mock api"
+  ],
+  "sideEffects": false,
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/grafana/grafana.git",
+    "directory": "packages/grafana-test-utils"
+  },
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "module": "src/index.ts",
+  "exports": {
+    ".": {
+      "import": "./src/index.ts",
+      "require": "./src/index.ts"
+    },
+    "./server": {
+      "import": "./src/server/index.ts",
+      "require": "./src/server/index.ts"
+    },
+    "./worker": {
+      "import": "./src/worker/index.ts",
+      "require": "./src/worker/index.ts"
+    },
```

Comment:
We pretty much have to export the server and worker behaviour separately, as they don't work in the opposing environment (e.g. server will crash if you try and load it in a browser context and vice versa)